### DISCUSSION TITLE
result grid > sample content returned in search result

### DISCUSSION THREAD

**Discussion by william-johnson-bloomreach** [bloomreach/limitless-ui-react#37] [Ideas]
Bloomreach Discovery can have content indexes.
Content items can be returned in search results.
See examples here
- https://pacific.bloomreach.com/home/search?_sq=decoration in tab "content"
- https://www.surlatable.com/search?prefn1=productType&prefv1=HardGood&q=pasta in tabs "classes" or "recipes"

**Comment by joerideg** [comment]
Hi William, if I understand correctly you can utilize the `searchType` attribute in the SearchBox for that. It can be one of 'product', 'bestseller', 'content' or 'categories'. See the [Storybook](https://bloomreach.github.io/limitless-ui-react/?path=/docs/components-searchbox--docs) page on SearchBox
